src/Str.php

```diff
@@ -60,11 +64,15 @@ public static function removeSuffix(string $value, string $suffix): string
      */
     public static function asClassName(string $value, string $suffix = ''): string
     {
-        $value = trim($value);
-        $value = str_replace(['-', '_', '.', ':'], ' ', $value);
-        $value = ucwords($value);
-        $value = str_replace(' ', '', $value);
-        $value = ucfirst($value);
+        $value = u($value)
```

Comment:
I'm not 100% sure, but I think this can be refactored as:

```php
return u($value)->camel()->title()->ensureEnd($suffix);
```

At least in the Symfony Docs show this:

```php
u('Foo: Bar-baz.')->camel()->title(); // 'FooBarBaz'
```

Reply:
seems ok